README.rst

```diff
@@ -1,48 +1,108 @@
 Pigeon
 ======
 
-AWS Lambda function that reacts to S3 object save events and sends crash ids to
-a configured RabbitMQ queue.
+AWS Lambda function that reacts to S3 object save events looking for new
+raw crashes in the S3 bucket. For each raw crash, it looks at the accept/defer
+annotation and sends the crash id to configured RabbitMQ queues for
+accepted crashes.
+
+The accept/defer annotation is the 7th-to-last character of the key::
+
+  v2/raw_crash/000/20180313/00007bd0-2d1c-4865-af09-80bc00180313
+                                                         ^
+
+* ``0`` - defer
+* ``1`` - accept
+* any other values are junk and ignored
 
 
 Quickstart
 ==========
 
-1. Clone the repo:
+1. `Install docker 1.10.0+ <https://docs.docker.com/engine/installation/>`_ and
+   `install docker-compose 1.6.0+ <https://docs.docker.com/compose/install/>`_
```

Comment:
I couldn't run the integration test with a docker-compose earlier than 1.20. We should update this requirement.